src/commonMain/kotlin/software/momento/kotlin/sdk/internal/UserHeaderInterceptor.kt

```diff
@@ -3,7 +3,7 @@ package software.momento.kotlin.sdk.internal
 import kotlin.jvm.Volatile
 
 internal class UserHeaderInterceptor(private val tokenValue: String) : io.grpc.ClientInterceptor {
-    private val sdkVersion = String.format("java:%s", this.javaClass.getPackage().implementationVersion)
+    private val sdkVersion = String.format("kotlin:%s", this.javaClass.getPackage()?.implementationVersion ?:  "unknown")
```

Comment:
can we distinguish between android and jvm versions?

Reply:
Yeah. I can pass that in from the grpc managers that instantiate it.